### PR TITLE
Restructuring global styles

### PR DESCRIPTION
**Pull request by kkatoch89**
## Description

Due to global styles conflict between styled components and material-ui, saved colour palette and fonts to JS variables. Also added a Layout component where I will be adding general styling and styling for different screen sizes.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓  | :bug: Bug fix              |
|  ✓  | :hammer: Refactoring       |

### After

globalStyles1

### CODE REVIEW

**Comment by ablizben** [comment]
Notistack notifications are no longer tiny - thank you, Karan! This looks great.

**Review by ablizben — APPROVED** [review]

**Review by dariatsvetkova — APPROVED** [review]
Beautiful! Thanks, @kkatoch89!

File: src/App.js [hunk 1]
```diff
@@ -11,6 +9,13 @@ import {
 import { db } from './lib/firebase';
 import { SnackbarProvider } from 'notistack';
 
+import {
+  GlobalStyles,
+  primary,
+  accent,
+  neutral,
+  Layout,
+} from './components/index';
```
Comment (luisaugusto) [inline src/App.js:18]: You don't need to write `/index` at the end of an import, Node will automatically check for a `index` file.

```suggestion
} from './components';
```

**Review by luisaugusto — APPROVED** [review]
Looks good, nice job setting up the global styles!

**Review by cafloyd — APPROVED** [review]
awesome, looks great @kkatoch89 !

**Pull request merged**